src/sources/shopify/webhookTransformations/serverSideUtlis.js

```diff
@@ -164,6 +165,25 @@ const handleCommonProperties = (message, event, shopifyTopic) => {
   return message;
 };
 
+/**
+ * Ensures message has an anonymousId, generating a random UUID if needed
+ * @param {Object} message rudderstack message object
+ * @param {Object} metricMetadata metric metadata object
+ * @returns {Object} message with anonymousId
+ */
+const ensureAnonymousId = (message, metricMetadata) => {
+  if (!message.anonymousId) {
+    message.anonymousId = uuidv4();
+    stats.increment('shopify_pixel_id_stitch_gaps', {
```

Comment:
who will monitor this stat?

Reply:
We are already doing this, we confirmed error from it only, this has various reason across the code, I have added another reason here. 

This is coming for the workspace we deployed for